examples/basic_example/basic_example.py

```diff
@@ -71,28 +71,49 @@
 from oemof.solph import EnergySystem
 from oemof.solph import Model
 from oemof.solph import buses
-from oemof.solph import components as cmp
+from oemof.solph import components as solver_components
 from oemof.solph import create_time_index
 from oemof.solph import flows
 from oemof.solph import helpers
 from oemof.solph import processing
 from oemof.solph import views
 
+BATTERY_STORAGE = "battery_storage"
```

Comment:
This is actually the label of the storage. It doesn't really matter if it's  battery or not, but I like defining the constant instead of having the string in the code several times.
```suggestion
STORAGE_LABEL = "battery_storage"
```